components/helptext/index.css

```diff
@@ -11,95 +11,75 @@
  * governing permissions and limitations under the License.
  */
 
-@media (forced-colors: active) {
-	.spectrum-HelpText {
-		--highcontrast-helptext-content-color-default: CanvasText;
-		--highcontrast-helptext-icon-color-default: CanvasText;
-
-		forced-color-adjust: none;
-
-		.spectrum-HelpText-validationIcon,
-		.spectrum-HelpText-text {
-			forced-color-adjust: none;
-		}
-	}
-}
-
 .spectrum-HelpText {
 	--spectrum-helptext-content-color-default: var(--spectrum-neutral-subdued-content-color-default);
 	--spectrum-helptext-icon-color-default: var(--spectrum-neutral-subdued-content-color-default);
 
-	&.spectrum-HelpText--sizeS {
-		--spectrum-helptext-min-height: var(--spectrum-component-height-75);
-		--spectrum-helptext-icon-size: var(--spectrum-workflow-icon-size-75);
-		--spectrum-helptext-font-size: var(--spectrum-font-size-75);
-		--spectrum-helptext-text-to-visual: var(--spectrum-text-to-visual-75);
-		--spectrum-helptext-top-to-workflow-icon: var(--spectrum-help-text-top-to-workflow-icon-small);
-		--spectrum-helptext-bottom-to-workflow-icon: var(--spectrum-helptext-top-to-workflow-icon);
-		--spectrum-helptext-top-to-text: var(--spectrum-component-top-to-text-75);
-		--spectrum-helptext-bottom-to-text: var(--spectrum-component-bottom-to-text-75);
-	}
-
-	&,
-	&.spectrum-HelpText--sizeM {
-		--spectrum-helptext-min-height: var(--spectrum-component-height-75);
-		--spectrum-helptext-icon-size: var(--spectrum-workflow-icon-size-100);
-		--spectrum-helptext-font-size: var(--spectrum-font-size-75);
-		--spectrum-helptext-text-to-visual: var(--spectrum-text-to-visual-75);
-		--spectrum-helptext-top-to-workflow-icon: var(--spectrum-help-text-top-to-workflow-icon-medium);
-		--spectrum-helptext-bottom-to-workflow-icon: var(--spectrum-helptext-top-to-workflow-icon);
-		--spectrum-helptext-top-to-text: var(--spectrum-component-top-to-text-75);
-		--spectrum-helptext-bottom-to-text: var(--spectrum-component-bottom-to-text-75);
-	}
-
-	&.spectrum-HelpText--sizeL {
-		--spectrum-helptext-min-height: var(--spectrum-component-height-100);
-		--spectrum-helptext-icon-size: var(--spectrum-workflow-icon-size-200);
-		--spectrum-helptext-font-size: var(--spectrum-font-size-100);
-		--spectrum-helptext-text-to-visual: var(--spectrum-text-to-visual-100);
-		--spectrum-helptext-top-to-workflow-icon: var(--spectrum-help-text-top-to-workflow-icon-large);
-		--spectrum-helptext-bottom-to-workflow-icon: var(--spectrum-helptext-top-to-workflow-icon);
-		--spectrum-helptext-top-to-text: var(--spectrum-component-top-to-text-100);
-		--spectrum-helptext-bottom-to-text: var(--spectrum-component-bottom-to-text-100);
-	}
+	/* Spacing */
+	--spectrum-helptext-min-height: var(--spectrum-component-height-75);
```

Comment:
I believe this medium help text needs the `component-height-100` token.

Reply:
To extend this comment, I noticed the large and extra large need some corrected min height tokens as well:
```
`&.spectrum-HelpText--sizeL {
  --spectrum-helptext-min-height: var(--spectrum-component-height-200); // instead of 100
}

&.spectrum-HelpText--sizeXL {
  --spectrum-helptext-min-height: var(--spectrum-component-height-300); // instead of 200
}
```